dev/AppAttach/AppAttachFramework/AppAttachAPI/Exceptions/Resources/error-codes.xml

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+<?xml version="1.0" encoding="utf-8"?>
+<errors>
+  <error code="AAA0004">
```

Comment:
What is AAA0004? Is this an HRESULT? Sorta?

You can define your own HRESULTs using FACILITY_ITF. See the `MDD_E_*` constants in [MsixDynamicDependency.h](https://github.com/microsoft/WindowsAppSDK/blob/main/dev/DynamicDependency/API/MsixDynamicDependency.h) and KOZANI_* in [Logging.h](https://github.com/microsoft/WindowsAppSDK/blob/develop/dev/Kozani/Common/Logging.h) for some examples.

NOTE: The Dynamic Dependencies ones are wrong and will be updated shortly. See the comment in `Logging.h` about 0-0x01FF vs 0x0200-0xFFFF and the link that's from for more details.